priv/schema.json

```diff
@@ -310,12 +310,17 @@
     },
 
     "tree_question":{
-      "question": {"$ref": "#/definitions/localized_string"},
-      "responses": {
-        "type": "array",
-        "minItems": 1,
-        "items": {"$ref": "#/definitions/tree_response"}
-      }
+      "type": "object",
+      "properties": {
+        "question": {"$ref": "#/definitions/localized_string"},
+        "responses": {
+          "type": "array",
+          "minItems": 1,
+          "items": {"$ref": "#/definitions/tree_response"}
+        }
+      },
+      "additionalProperties": true,
```

Comment:
It's not safe to allow additionalProperties, let's better set it to false and enumerate every property that the object has. Like the id for instance, that should also be required.